int32 binId
float32 x
float32 y
float32 z
---
int32 collision_num